### ISSUE TITLE
Instructions use wrong config file

### ISSUE THREAD

**Issue by KorakSilvercloud** [BluGeni/Pandora-Record#1]
Doesn't look like this has been active in years, but just in case someone else stumbles across it. The instructions tell you to use the wrong config file. Put your configuration info into the config.json and things should work fine. 

Hopefully this saves someone the couple hours I spent scratching my head trying to figure that out :)

**Comment by BluGeni** [comment]
Is this the part you are referring to?
`sudo nano ~/.config/pianobar/config`

Let me know what to change and ill fix it or open a pull request.

**Comment by bru73f0rc3** [comment]
Yep, the directions state 3 lines to modify in record-pandora.conf when it's 4 lines in config.json. :)

**Comment by bru73f0rc3** [comment]
also the readme states to change the default-sample-rate in /etc/pulse/daemon.conf to "480000", should be 48000
